src/backend/state.rs

```diff
@@ -98,6 +99,12 @@ impl OpenGlState {
             // SAFETY: `source_scale` is declared as a `uvec2`
             gl::Uniform2ui(uniforms.source_scale, source_scale.0, source_scale.1);
 
+            let source_rotation = 0;
+            // An angle of 0 means identity matrix
+            // SAFETY: `source_rotation` is declared as a `mat2`
+            let rot_mat: [[f32; 2]; 2] = [[1.0, 0.0], [0.0, 1.0]];
+            gl::UniformMatrix2fv(uniforms.source_rotation, 1, gl::FALSE, rot_mat[0].as_ptr());
```

Comment:
```suggestion
            gl::UniformMatrix2fv(uniforms.source_rotation, 1, gl::FALSE, rot_mat.as_ptr().cast::<f32>());
```
using `&array[0]` to get a pointer to the whole array is probably unsound, see https://rust-lang.zulipchat.com/#narrow/stream/136281-t-lang.2Fwg-unsafe-code-guidelines/topic/ptr.20to.20array.20using.20array.20indexing for more details.